NOTICE.txt

```diff
@@ -133,55 +138,24 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
----
-
-## @types/country-list
```

Comment:
We're probably fine to remove these since the type definitions are only used in development, but since we're removing this, does it mean that we're ignoring everything that's set as a dev dependency in the package.json? I recently realized that, even if something is a dev dependency, some of its code might get compiled into the app like with Webpack which injects some bootstrapping code into the app. In that case, we might want to actually include Webpack in here

Reply:
At the moment we do not include any dependency from dev dependency list. If that is needed we can include all of them. Or we may add them if they exists in dev dependency list(For that feature, we will need full package name of those).

Reply:
Dev dependencies are not generally needed for NOTICE.txt. I don't know how to handle the case Harrison describes above.

Reply:
@amyblais We can define predefined possible package list, and if they are referred in devDependencies we can add those to the Notice.txt. All we need is full list of dependencies. At the moment only weblate is known. 

We will define a config property similar to below
INCLUDE_DEV_DEPS=weblate

Application will scan devDependencies and if any dev dependency defined at `INCLUDE_DEV_DEPS` property it will add to the Notice.txt.

@hmhealey Do you know is there any other dependency which we consider at devDependencies ?